cwms-data-api/src/main/java/cwms/cda/api/watersupply/WaterPumpDisassociateController.java

```diff
@@ -65,8 +65,8 @@ public WaterPumpDisassociateController(MetricRegistry metrics) {
             @OpenApiParam(name = PUMP_TYPE, required = true, type = PumpType.class,
                     description = "The type of pump to be disassociated from the contract."
                             + " Expected values: IN, OUT, OUT BELOW"),
-            @OpenApiParam(name = METHOD, type = boolean.class, required = true,
-                    description = "Whether to delete the associated accounting data.")
+            @OpenApiParam(name = METHOD, type = boolean.class,
```

Comment:
I think this is a nomenclature thing, METHOD isn't the right word.  It should be something like "Delete Accounting" or something, cause it's just a boolean not a DeleteMethod.

Reply:
isn't that the distinction between Delete Key and Delete Data? changing the query parameter is fine too

Reply:
How about renaming the query parameter to DELETE_ACCOUNTING ("delete-accounting")?

Reply:
that's fine, unless you can get swagger to pick only two of the three DeleteMethod enums

Reply:
I hadn't seen your comment from yesterday on this, can you explain why are you opposed to using a boolean for this vs the DeleteMethod piece?

Reply:
Just realized we never finished this conversation.  I can see the benefit of that.  Would it look something like DELETE_KEY == false and DELETE_DATA || DELETE_ALL == true?  Or are you thinking just DELETE_KEY and DELETE_DATA are acceptable?  This would be if the naming stuck with METHOD, since that's also consistent with the other naming usages.

Reply:
Hmm that's a good point. If you specified `DELETE_DATA`, then I'd expect the accounting to get deleted, but the pump not disassociated? I was envisioning DELETE_KEY and DELETE_ALL would be the only two options, but I don't even know if swagger can do that. 

Sounds to me like DeleteMethod doesn't make sense here and a simple boolean is sufficient?

Reply:
Yeah...being that you have 2 options instead of 3, I think that's more appropriate to be a boolean.  I can see the benefit of it being consistent, but that would throw in some additional expected behaviors that's not supported.

Reply:
@zack-rma please move forward with updating this to use a boolean, use the name you provided for DELETE_ACCOUNTING and let me know when you're done so I can re-review.